templates/test/downloads.html.ep

```diff
@@ -4,7 +4,7 @@
         % for my $resultfile (@$resultfiles) {
             <li>
                 % if($resultfile =~ /video.ogv$/) {
-                    %= link_to url_for('test_file', testid => $testid, filename => 'video.ogv') => begin
+                    %= link_to url_for('video', testid => $testid) => begin
```

Comment:
That makes it harder to download the video or to copy the link to open it with mpv for example. I would preserve the old link and add the player only additionally.

Reply:
yes, good idea. After all the "logs & assets" tab should provide the files for download first-hand